crates/blockchain/blockchain.rs

```diff
@@ -1217,6 +1217,24 @@ impl Blockchain {
                 "".to_string()
             };
             info!("{}{}", base_log, extra_log);
+            if as_gigas > 0.0 {
```

Comment:
I added this log line to have an easy way to copy/paste metrics into a csv or excel sheet to calculate averages.